CHANGELOG.md

```diff
@@ -1,4 +1,9 @@
 
+#### [3.2.2](https://github.com/neilang/NAMapKit/tree/v3.2.2) (5/2/2016)
+
+* [#65](https://github.com/neiland/NAMapKit/pull/65) - Exposed: `ImageView` to assist with getting accurate CGPoints when `NAMapKit` scrollview scale != 1.0f (ex; CGPoint p = [gesture locationInView:mapView.imageView];) - [@multinerd](https://github.com/multinerd).
```

Comment:
This is TMI, you might want to add something to he README, although this seems awfully specific to your problem. Maybe you want to expose a higher level method too?